docs/deployment-examples/prometheus/prometheus.yml

```diff
@@ -0,0 +1,14 @@
+# Sample Prometheus config
+# This assumes that your Prometheus instance can access this application on localhost:8080
+
+global:
+  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+scrape_configs:
+  - job_name: 'spring boot scrape'
+    metrics_path: '/actuator/prometheus'
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:8080']
```

Comment:
Do you need to set the global scrape_interval if you will override it in the scrape configs?

Reply:
its just something i've seen in multiple other docker files (including this one that was copied). useful if you have multiple jobs and dont want to specify the scrape_interval for each one.  
I can get rid of it if you'd like